hwdb.d/60-sensor.hwdb

```diff
@@ -450,6 +450,12 @@ sensor:modalias:platform:lis3lv02d:dmi:*svn*Hewlett-Packard*:*
 sensor:modalias:platform:lis3lv02d:dmi:*svn*HP*:*
  ACCEL_LOCATION=base
 
+# HP Elitebook 2760p
+# Physicaly accel is in base, but 2760p is 2 in 1 transformer, and accel can be used in tablet mode to rotate screen
```

Comment:
Physicaly → Physical?

Reply:
FWIW, this is exactly the case where an adverb is needed, i.e. this should have been "physically". In this sentence, an adverb would apply to "is", but an adjective would apply to "accel". I guess we should use a proper word, not "accel" too.